### DISCUSSION TITLE
What's the best practice for adding messages in `on_fetch`?

### DISCUSSION THREAD

**Discussion by xq114** [xmake-io/xmake#1831] [General]
目前查找包如果走xmake-repo安装会输出如下提示信息
```
checking for xmake::glew ... glew 2.2.0
```
但对于通过`on_fetch`查找的包默认不会输出任何提示。在`on_fetch`中加入查找的信息是必须的，但是信息中`glew 2.2.0`颜色受xmake本身theme控制，直接print写死感觉不太对，这里的best practice是怎样的？

**Comment by waruqi** [comment]
我加上了

```
checking for xmake-repo::xxx ... xxx 1.0
```
